apps/coscrad-frontend/src/components/resources/utils/render-media-length-in-minutes-cell.tsx

```diff
@@ -0,0 +1,5 @@
+export const renderMediaLengthInMinutes = (length): JSX.Element => {
```

Comment:
For flexibility, we might want to inject a second arg, a formatter, that defaults to the format you've picked here.

Reply:
Ok right, that would make sense.